src/poliastro/twobody/orbit.py

```diff
@@ -580,6 +580,32 @@ def geostationary(
         altitude = geo_radius - attractor.R
         return cls.circular(attractor, altitude)
 
+    @classmethod
+    @u.quantity_input(a=u.m, ecc=u.one, w=u.rad / u.s, period=u.s)
+    def heliocentric(cls, attractor, a, ecc, w=None, period=None):
+        if w is None and period is None:
+            raise ValueError("At least one among w or period must be passed")
+
+        if w is None:
+            w = 2 * np.pi / period
+
+        if a is None:
+            raise ValueError("The value of semi major axis is required")
+        mean_motion = np.sqrt(attractor.k / np.cube(a))
+        h = a * (1 - ecc ** 2)
```

Comment:
Could you please document what's `h` and where does the `theta` and `inc` values come from?

Reply:
May I suggest that the variable `mean_motion` could be called `n`? Since mean_motion is one of the propagation methods that `poliastro`includes, it could lead to confusion or coding errors in the future.

Reply:
`h` is the semi-latus rectum. `inc` is the cosine of inclination for a given value of semi-major axis, eccentricity, and time period (in radians). `theta` is the original angle in degrees.
Also, I will replace `mean_motion` by `n`.